.github/workflows/create-demo-clusters.yml

```diff
@@ -416,7 +441,7 @@ jobs:
     needs: [properties, start-secured-cluster]
     runs-on: ubuntu-latest
     env:
-      SECURED_CLUSTER_NAME: ${{ inputs.cluster-with-real-load-name }}
+      INFRA_NAME: ${{ inputs.cluster-with-real-load-name }}
```

Comment:
Now that the `start-kube-burner.sh` script uses the name of the cluster, since it is part of the UUID for the kube-burner run, the name of the environment variable needs to be the same for the central and secured clusters.